src/BlockTypes/ProductGalleryPager.php

```diff
@@ -78,11 +87,11 @@ private function render_pager( $pager_display_mode ) {
 	 */
 	private function render_digits_pager() {
 		return sprintf(
-			'<ul class="wp-block-woocommerce-product-gallery-pager__pager">
-				<li class="wp-block-woocommerce-product-gallery__pager-item is-active">1</li>
-				<li class="wp-block-woocommerce-product-gallery__pager-item">2</li>
-				<li class="wp-block-woocommerce-product-gallery__pager-item">3</li>
-				<li class="wp-block-woocommerce-product-gallery__pager-item">4</li>
+			'<ul class="wc-block-product-gallery-pager__pager">
```

Comment:
I am adding a comment here, but it is for line 51. We should remove `woocommerce` class. What do you think?

![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/a0d06eb5-fefe-4284-bb09-f954e7415d2f)